apps/vr-tests/src/stories/LinkNext.stories.tsx

```diff
@@ -3,41 +3,31 @@ import * as React from 'react';
 import Screener, { Steps } from 'screener-storybook/src/screener';
 import { storiesOf } from '@storybook/react';
 import { FabricDecorator } from '../utilities';
-import { Customizer, Link, ILinkStyleProps, ThemeProvider } from '@fluentui/react-next';
-
-const customStyles = { root: { background: 'lightblue' } };
-const themedStyles = (props: ILinkStyleProps) => ({
-  root: {
-    background: props.theme.palette.greenLight,
-  },
-});
-const scopedSettings = { Link: { styles: customStyles } };
+import { Link } from '@fluentui/react-next';
 
 storiesOf('Link Next', module)
   .addDecorator(FabricDecorator)
   .addDecorator(story => (
-    <ThemeProvider>
-      <Screener
-        steps={new Steps()
-          .snapshot('default', { cropTo: '.testWrapper' })
-          .executeScript(
-            "document.getElementsByClassName('testWrapper')[0].classList.add('ms-Fabric--isFocusVisible')",
-          )
-          .executeScript("document.getElementsByClassName('ms-Link')[0].focus()")
-          .snapshot('focus', { cropTo: '.testWrapper' })
-          .executeScript(
-            "document.getElementsByClassName('testWrapper')[0].classList.remove('ms-Fabric--isFocusVisible')",
-          )
-          .hover('.ms-Link')
-          .snapshot('hover', { cropTo: '.testWrapper' })
-          .click('.ms-Link')
-          .hover('.ms-Link')
-          .snapshot('click', { cropTo: '.testWrapper' })
-          .end()}
-      >
-        {story()}
-      </Screener>
-    </ThemeProvider>
+    <Screener
+      steps={new Steps()
+        .snapshot('default', { cropTo: '.testWrapper' })
+        .executeScript(
+          "document.getElementsByClassName('testWrapper')[0].classList.add('ms-Fabric--isFocusVisible')",
+        )
+        .executeScript("document.getElementsByClassName('ms-Link')[0].focus()")
+        .snapshot('focus', { cropTo: '.testWrapper' })
+        .executeScript(
+          "document.getElementsByClassName('testWrapper')[0].classList.remove('ms-Fabric--isFocusVisible')",
+        )
+        .hover('.ms-Link')
+        .snapshot('hover', { cropTo: '.testWrapper' })
+        .click('.ms-Link')
+        .hover('.ms-Link')
```

Comment:
Maybe we should remove this second `.hover`?

Reply:
I am resolving this fix in this PR https://github.com/microsoft/fluentui/pull/14809/files#diff-cc28f495c7f67ad88dc920d1100b79f0 to avoid needing re-run the build :p